src/frontend/src/lib/components/transactions/Transaction.svelte

```diff
@@ -41,7 +41,7 @@
 
 		<svelte:fragment slot="amount">
 			{#if nonNullish(amount)}
-				<Amount {amount} decimals={token.decimals} />
+				<Amount {amount} decimals={token.decimals} formatPositiveAmount />
```

Comment:
Are you sure the `+` sign is always applicable? Like it's the same regardless of the type of transaction?

Reply:
Good question! According to code we have 7 types of transactions:

- send --> negative
- receive --> positive
- withdraw --> positive --> conversion from ck-token
- deposit --> negative --> conversion to ck-token
- approve --> null (so no new format)
- mint --> positive --> conversion from twin token
- burn --> negative --> conversion to twin token

So, I imagine that all the positive-amount cases fit the new format, right?